recipes/sfml/all/conanfile.py

```diff
@@ -51,6 +52,10 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def build_requirements(self):
+        if Version(self.version) >= "3.0.0":
+            self.tool_requires("cmake/[>=3.24]")
```

Comment:
3.24 is only required on Windows. All other OSes only require 3.22.

Reply:
Fixed in a6b8e39528c02a2aa8f3397f7be3ba7814d9159c